Gemfile

```diff
@@ -14,6 +14,8 @@ gem 'aws-sdk-s3', '~> 1'
 gem 'dotenv-rails', '~> 2.7'
 gem 'rails', '~> 6.0'
 
+gem 'bootsnap', '~> 1.16'
```

Comment:
why? how would this be useful in production?

Reply:
I moved it out of the development group because it's being required [here](https://github.com/3scale/porta/pull/3456/files#diff-441bbd2acf3b2eab84a98d917f574109eafd6c54cbce74a2379487f9f67920a2R4) and that's the reason the community image fails to build.

But you're right, the best solution is to not call `require 'bootsnap/setup'` when rails environment is `:production`.

Reply:
What's the problem though having it in production?
I understand that it's Rails who adds this by default, so I guess there might be some benefit for production as well?

Reply:
I think it's better to not have it in production because it uses a `/tmp` folder in disk as cache, and that cache is never erased. So we need to maintain that folder, either keeping it in the container filesystem or mounting a volume for it, in case the container is read-only. And it's only advantage is increasing boot speed by a few seconds, which we won't really notice since the whole rollout process in OCP takes like 5-7 minutes.

Reply:
The way it is required is incorrect as far as I remember. I think it never properly worked. I had it on the back of my mind to some day look at it. If necessary, stop requiring it at all and at some point, we will see how to properly enable it.

Reply:
I don't know. It seems to me the porta boots much faster locally in Rails 6 than Rails 5. So it should be working fine.